debian/changelog

```diff
@@ -1,5 +1,6 @@
 cloud-init (21.4-82-g0d364a29-0ubuntu1~18.04.1) UNRELEASED; urgency=medium
 
+  [ Brett Holman ]
```

Comment:
Since we don't really call out individual contributions in debian/changelog for Canonical contributors in cloud-init, let's redact the separate authorship sections here and just add your one liner to the top of the changelog. Also generally because d/changelog entries in cloud-init are so large at SRU time, it's been requested that our team consolidate any package related changes  at the top of the changelog. So make sure to pull your d/cloud-init.templates entry up to the top of the changelog section.